server/sonar-server/src/main/java/org/sonar/server/qualitygate/QualityGates.java

```diff
@@ -71,7 +76,7 @@
   private final Settings settings;
 
   public QualityGates(QualityGateDao dao, QualityGateConditionDao conditionDao, MetricFinder metricFinder, PropertiesDao propertiesDao, ComponentDao componentDao,
-                      MyBatis myBatis, UserSession userSession, Settings settings) {
```

Comment:
very minor, but I wonder what is the best indentation. I have the same configuration, but I find the old one better